porsche-digital-lab-berlin-blockchain/catbox-contracts
# Better deployments

## What does this pr do?
* saves a file named `deployments.json` to root which is easier to use in other projects due to being saved by network name rather than networkId. Also includes `deploymentBlock` field which is useful for deciding on where to start looking at past events (you dont need to look for events before a contract existed... duh)
* updates `.eslintrc.js` to reflect the global web3 instance which is put in place by truffle
* adds another function to `hashRegistry` in order to allow for sharing encrypted files
* fixes a security issue raised by github via `resolutions` in `package.json`

## How to test?
If you want, you can run a local instance of loom dappchain:
```sh
yarn loom:init
yarn gen:loom-key
yarn loom:run
```
check the `deployments.json` file... specifically the `loom_dapp_chain` field...

then try deploying to the local instance:
```sh
yarn deploy:reset
```
you should then see that the previously mentioned field has changed reflecting the update.
